json/test/positive/omitted-state.json

```diff
@@ -0,0 +1,11 @@
+{
+  "flags": {
+    "myBoolFlag": {
+      "variants": {
+        "on": true,
+        "off": false
+      },
+      "defaultVariant": "on"
```

Comment:
`defaultVariant` should be either `true` or `false`, right? [1]

[1] - https://github.com/open-feature/schemas/issues/80#issuecomment-1431937964

Reply:
Ah I misinterpreted that comment, I think you're right, thanks. I'll update to use `"true": true, "false": false`

Reply:
Amended.